### DISCUSSION TITLE
GoPro Hero 8 black file reduce chapter size

### DISCUSSION THREAD

**Discussion by gustavoneto05** [gopro/labs#610] [Q&A]
I've done a lot of research and haven't been able to find a solution so far. I have a GoPro Hero 8 Black and I wish there was a setting to reduce the size of each recorded chapter. In fact, I wanted my GoPro to record videos every 1 minute continuously.
Does anyone have any ideas about this?

**Comment by dnewman-gpsw** [comment]
The idea for smaller chapters was added after HERO8, HERO11, Mini and 12 can set smaller or larger chapter sizes.
